README.md

```diff
@@ -58,3 +58,23 @@ See `CONTRIBUTING.md`
     - `git tag -s` the release
     - `git push --tags`
     - `cabal sdist` and `cabal upload`
+
+## travis
+
+`.travis.yml` is generated using `make-travis-yml` tool, in
+[multi-ghc-travis](https://github.com/haskell-hvr/multi-ghc-travis) repository.
+
+To regenerate the script use (*note:* atm you need to comment `doc/cookbook/` packages).
+
+```
+runghc ~/Documents/other-haskell/multi-ghc-travis/make_travis_yml_2.hs regenerat
+```
```

Comment:
Hmm, so the idea is that we'd never manually change the `.travis.yml` file?

Reply:
yes